cmd/homi/setup/flags.go

```diff
@@ -65,7 +65,7 @@ var (
 	numOfCNsFlag = cli.IntFlag{
 		Name:  "cn-num",
 		Usage: "Number of consensus nodes",
-		Value: 4,
+		Value: 0,
```

Comment:
This makes sense for service chains, but not for main chain.
What about leaving this 4 and setting it to zero when `--service-chain` is enabled?

Reply:
For docker-compose, if we generate docker-compose like below, the default cn-num makes 4 CN nodes.
```
homi setup docker --no-grafana --scn-num 1
```

So I wanted to remove unintended nodes.